Rubberduck.Core/UI/Inspections/InspectionResultsControl.xaml

```diff
@@ -22,7 +23,9 @@
             <codeInspections:InspectionImageSourceConverter x:Key="InspectionIconConverter" />
             <codeInspections:QuickFixImageSourceConverter x:Key="QuickFixIconConverter" />
 
-            <Style x:Key="IconMargin" TargetType="Image">
+			<converters:GroupingGridLocationConverter x:Key="GroupingGridLocationConverter" />
```

Comment:
This converter should be reused in other grids that allow grouping by location (or that *should eventually* allow it) - thinking this static resource could be in a resource dictionary, making it reasier to reuse without needing to re-declare.